logic/transaction.js

```diff
@@ -409,277 +409,285 @@ class Transaction {
 	 * @todo Add description for the params
 	 */
 	verify(transaction, sender, requester, cb, tx) {
-		let valid = false;
-		let err = null;
-
 		if (typeof requester === 'function') {
 			cb = requester;
 		}
```

Comment:
@ManuGowda We are removing such practices of mutating function parameters, wherever we refactor the code. So please remove this if block and call the `verify` with the exact number of params.